README.md

```diff
@@ -135,7 +135,35 @@ $this->hasAttribute('body.user_id');
 $this->getAttributeValue('body.user_id');
 ```
 
+#### Attributes
 
+You can use attributes in an action handler to get data from a request:
+
+```php
+use Psr\Http\Message\ResponseInterface;
+use Yiisoft\RequestModel\Attribute\RouteParam;
+
+final class SimpleController
+{
+    public function action(#[RouteParam('id')] int $id, #[ReqAttribute('foo')] $attribute,): ResponseInterface
+    {
+        echo $id;
+        //...
+    }
+}
+```
+
+Attributes are also supported in closure actions.
+
+There are several attributes out of the box:
+
+| Name          | Source                    |
+|---------------|---------------------------|
+| ParsedBody    | Parsed body of request    |
+| QueryParam    | Query parameter of URI    |
+| ReqAttribute  | Attribute of request      |
```

Comment:
How about using full name, i.e. `RequestAttribute`? Because now it could be confused with "required attribute".